spec.emu

```diff
@@ -263,34 +257,5 @@ copyright: false
       1. Throw a *TypeError* exception.
     </emu-alg>
   </emu-clause>
-
-  <emu-clause id="sec-getoption" type="abstract operation">
-    <h1>
-      GetOption (
```

Comment:
why not use `GetOption` for the `mode` option? it would handle the default in a single step.

Reply:
`GetOption` (taken from 402) coerces the value based on the third parameter. We have a normative convention to no longer do that.

Reply:
Sure, but that suggests we should enhance GetOption to skip coercing, since we'll be using GetOption in Temporal anyways.

Reply:
Yeah sure, but again, that's in 402, so not really this proposal's problem. We'll have to figure out how to handle that editorially when merging temporal, since its coercion behaviour is presumably not changing to match our new conventions.

Reply:
Fair, but its coercion behavior should definitely be changing to match our new conventions - it hasn't shipped on the web yet so there's no reason not to do so.

Reply:
You're going to have to bring that up with the temporal people, not me. Also, they're not the only users. GetOption, with its current coercing behaviour, is used pretty extensively throughout 402 today: https://tc39.es/ecma402/#sec-getoption